node.js

```diff
@@ -258,5 +265,24 @@ module.exports = {
   clearCaches: function clearCaches () {
     filenessCache = { }
     configCache = { }
+  },
+
+  testCanIUse: function testCanIUse (latestVersion) {
+    var now = Date.now()
+    var digitsDiff = String(now).length - String(latestVersion).length
+    if (digitsDiff) {
+      latestVersion = padToEnd(latestVersion, digitsDiff)
+    }
+    var diffDates = now - latestVersion
+    diffDates = diffDates / 1000 / 60 / 60
```

Comment:
It will be better to have `MS_TO_UPDATE_CANIUSE` instead of `DAYS_TO_UPDATE_CANIUSE`, so you will not call calculation every time.